icons/circle/48/towerfall-ascension.svg

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Can you please re-save the icons as optimized using those options? https://github.com/numixproject/numix-core/wiki/Optimise-Options
The XML declaration and some unneeded groups can be removed :)

Reply:
Hmm...
* Isn't the build process contain svg minification?
  You know, I usually keep some groups while developing to help the readability of the code.
  So if the process will minify the svg-s inside the package, I would rather keep the groups.
* Also, isn't the xml declaration needed for some renderers (correct me if I am wrong)?

Reply:
The build process doesn't contain any SVG minification for now, and I don't think it will be the case (@Foggalong ?). We are trying to get some optimizations tests, so the icons can be tested as "optimized" before being merged into the core repository. Minifying the SVG files for each release will take a much longer time (around 10 minutes per theme...)

As far as I know, it's the SVG declaration that is needed 
```xml
<svg version="1.1" viewBox="0 0 48 48" xmlns="http://www.w3.org/2000/svg">
```

Reply:
For the readability part, you can keep the icons as they're until it's approved by @palob & @Foggalong and then you can re-optimize it if that will help 👍

Reply:
@bil-elmoussaoui @matemeszaros The build process doesn't doesn't no because that would only give a size benefit to packaging repos, and not this repo also.